docs/topics/js/js-ir-migration.md

```diff
@@ -214,4 +170,39 @@ kotlin {
         binaries.executable()
     }
 }
+```
+
+## Additional troubleshooting tips when working with the Kotlin/JS IR compiler
+
+These hints may help you when troubleshooting problems in your projects using the Kotlin/JS IR compiler.
+
+### Make boolean properties nullable in external interfaces
+
+**Issue**: when you call `toString` on a `Boolean` from an external interface, you're getting an error like `Uncaught TypeError: Cannot read properties of undefined (reading 'toString')`. JavaScript treats the `null` or `undefined` values of a boolean variable as `false`. If you rely on calling `toString` on a `Boolean` that may be `null` or `undefined` (for example when your code is called from JavaScript code you have no control over), be aware of this:
+
+```kotlin
+external interface SomeExternal {
+    var visible: Boolean
+}
+
+fun main() {
+    val empty: SomeExternal = js("{}") as SomeExternal
```

Comment:
`as SomeExternal` is redundant

Reply:
...finger must've slipped when copying this snippet from my sample project, d'oh 😱